core/indexer/processTransactions.go

```diff
@@ -248,6 +251,33 @@ func (tdp *txDatabaseProcessor) groupNormalTxsAndRewards(
 	return transactions, rewardsTxs
 }
 
+func (tdp *txDatabaseProcessor) setTransactionSearchOrder(transactions map[string]*Transaction, shardId uint32) map[string]*Transaction {
+	currentOrder := 0
+	shardIdentifier := tdp.createShardIdentifier(shardId)
+
+	for _, tx := range transactions {
```

Comment:
The iteration is done over a map. This will not ensure the same order each time. Does it matter?

Reply:
You are right. But we don't care. The only thing we are interested in here is to have a tie breaker for the front end where the shard id is the same and the (round) timestamp is the same

Reply:
ok